README.md

```diff
@@ -1,3 +1,89 @@
-# gz-beat-shipper
+# beatshipper
 
 Since there’s no way to send GNU zip files through filebeat, this service will be responsible for checking if there are new .gz files based on a path that we’ll explode using globbing, decompress them, send them using the filebeat service with the provided configuration and store them in a local file registry.
+
+It can be used also for not compressed files.
```

Comment:
'It can **also** be used for **non-compressed** files.'

Reply:
Oh, thanks. Change done. I knew the B2.2 of the English school wasn’t good at all :p